app/models/foreman_openscap/arf_report.rb

```diff
@@ -192,5 +201,22 @@ def self.references_links(references)
       end
       html_links.join(', ')
     end
+
+    def self.check_msg_for_changes(msg, incoming_data)
+      mapping = { :severity => :severity,
+                  :description => :description,
+                  :rationale => :rationale,
+                  :scap_references => :references,
+                  :value => :title }
+      mapping.each do |msg_attr, data_key|
+        msg.public_send("#{msg_attr}=".to_sym, incoming_data[data_key]) unless msg.public_send(msg_attr) == incoming_data[data_key]
```

Comment:
that does not seem like standard behavior, tested in console

```ruby
irb(main):005:0> m = Message.last
1 row in set
irb(main):006:0> m.value
=> "content changed '{md5}f737ea43ce072a92d9efe885aa393712' to '{md5}a3991e3a43da21c1a2ba9d349dcb9ffb'"
irb(main):007:0> m.value = "content changed '{md5}f737ea43ce072a92d9efe885aa393712' to '{md5}a3991e3a43da21c1a2ba9d349dcb9ffb'"
=> "content changed '{md5}f737ea43ce072a92d9efe885aa393712' to '{md5}a3991e3a43da21c1a2ba9d349dcb9ffb'"
irb(main):008:0> m.changed?
=> false
```

is there some specific attribute for which it does not work? I guess the example you provided would not work, message does not have `title` attribute. If you're afraid this rails behavior could change in future, a test to ensure it's working this way would be good, but I doubt it will change how it works.